DEVELOPMENT.md

```diff
@@ -0,0 +1,97 @@
+## Run services with `go run`
+
+This section explains how to run some of the services using `go run` instead of the Docker setup to facilitate development.
+
+### Running `sda-download` with `go run`
+1. Bring up all SDA services with the S3 backend and populate them with test data by running the following command in the root folder of the repository:
+```sh
+make integrationtest-sda-s3-run 
+```
+
+2. Change to the folder `sda-download` and then create the credentials using: 
+```sh
+pushd dev_utils
+bash make_certs.sh
+chmod 600 certs/*-key.*
+popd
+```
```

Comment:
```suggestion
```

Certificates are not needed since it specifically says to use the `no_tls` config file